src/app/components/common/navbar/navbar.component.html

```diff
@@ -1,7 +1,48 @@
-<div class="row align-items-center h-100">
-  <div class="col-6 mx-auto text-center">
-    <div class="spinner-border" style="width: 3rem; height: 3rem" role="status">
-      <span class="sr-only">Loading...</span>
+<!--navigation bar-->
+<nav class="navbar navbar-expand-lg navbar-light bg-white navigationBar">
+  <div class="container-fluid">
+    <button aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation"
+            class="navbar-toggler"
+            data-bs-target="#navbarNavDropdown" data-bs-toggle="collapse" type="button">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+      <div class="row col-12 content-row">
+        <div class="col-lg-4 col-sm-12 navgLinks">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a aria-current="page" class="nav-link active" href="#">Feed</a>
```

Comment:
add more space using px-4 class for navlinks

Reply:
done